readinglist/__init__.py

```diff
@@ -53,6 +55,7 @@ def set_auth(config):
 
     config.set_authorization_policy(authz_policy)
     config.set_authentication_policy(authn_policy)
+    config.set_default_permission('readwrite')
```

Comment:
I give by default the most needed rights so that developers can ask for less permissions if needed.